tests/LINEBot/NarrowCastTest.php

```diff
@@ -85,6 +86,13 @@ public function testSendNarrowcast()
                         ->setNot(
                             AudienceRecipientBuilder::builder()
                                 ->setAudienceGroupId(21234567890)
+                        ),
+                    RedeliveryRecipientBuilder::builder()
```

Comment:
Please add assertion/tests in `$testRunner` to verify the data you appended.

Reply:
@Vaduz thank you for reviewing. added test assert.
https://github.com/line/line-bot-sdk-php/pull/306/commits/daebefce4e04f78cc329a5b0bef6997852cf9668